server/src/main/scala/org/http4s/server/middleware/VirtualHost.scala

```diff
@@ -20,53 +21,51 @@ object VirtualHost {
     * filled in, if possible, using the request Uri or knowledge of the
     * security of the underlying transport protocol.
     */
-  final case class HostService[F[_]](
-      @deprecatedName('service) routes: HttpRoutes[F],
+  final case class HostService[F[_], G[_]](
+      @deprecatedName('service) http: Kleisli[F, Request[G], Response[G]],
```

Comment:
Do we want to use the `Http[F, G]` alias instead of `Kleisli[F, Request[G], Response[G]]`?  I'm inclined toward yes.